astro.config.mjs

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from 'astro/config';
+
+import node from '@astrojs/node';
+
+// https://astro.build/config
+export default defineConfig({
+	output: 'server',
+	adapter: node({
+		mode: 'standalone',
+	}),
+	vite: {
+		ssr: {
+			noExternal: ['path-to-regexp'],
```

Comment:
Is "path-to-regexp" a placeholder? shouldnt it be set?

Reply:
this is a workaround for an [issue in astro atm](https://docs.astro.build/en/guides/integrations-guide/node/#syntaxerror-named-export-compile-not-found)